src/lib.rs

```diff
@@ -139,6 +139,11 @@ impl<Key: PartialOrd + Ord + Clone, Value: Clone + Eq + Hash> Accumulator<Key, V
     pub fn set_quorum_size(&mut self, new_size: usize) {
         self.quorum = new_size;
     }
+
+    /// Returns the current value for `quorum`.
+    pub fn get_quorum_size(&self) -> usize {
```

Comment:
Hi, thanks for the PR, really nice. I wonder though if it is more idiomatic to call this method (getter) just `quorum_size`. I will ask @afck  what he thinks as well. I generally use set for setters and the item name alone for getters.